RGB LED Control V2.0/code/Library/std_types.h

```diff
@@ -0,0 +1,36 @@
+/*
+ * 	File:						std_types.h
+ *	Module:					library
+ *	Description: 		header file 
+ * 	Author: 				HAZEM ASHRAF
+ */
+
+
+#ifndef STD_TYPES_H_
+#define STD_TYPES_H_
+
+typedef enum {FALSE,TRUE}	boolean;
+
+#define LOGIC_HIGH        (1u)
+#define LOGIC_LOW         (0u)
+
+#define NULL_PTR    ((void*)0)
+	
+typedef enum{
```

Comment:
Each driver should have its own enum error state

Reply:
Done, you can check it now

Reply:
Great, well done.👌🏻

Reply:
thank you